src/presentation/http/policies/userInTeam.ts

```diff
@@ -30,4 +30,13 @@ export default async function userInTeam(request: FastifyRequest, reply: Fastify
   if (creatorId !== userId) {
     return await reply.forbidden();
   }
+
+  const { teamMemberId } = request;
```

Comment:
you need to add this property to request (if it is required for most requests), or access domain service from there